lib_test/tests.ml

```diff
@@ -158,6 +170,11 @@ let suite = [
     "aligned to 512"  , `Quick, check_alignment 512;
     "aligned to 0"    , `Quick, check_alignment_zero;
     "aligned to large", `Quick, check_alignment_large;
+  ];
+  "rev", [
+    "empty", `Quick, rev_empty;
+    "len = 1", `Quick, rev_len_1;
+    "len = 5", `Quick, rev_len_5;
```

Comment:
any chance of a len=0 test for completeness?

Reply:
That's what the `rev_empty` test does. Do you mean it should be renamed?

Reply:
missed that, thanks!